source/MRViewer/MRFileDialog.cpp

```diff
@@ -422,13 +325,24 @@ std::vector<std::filesystem::path> gtkDialog( const FileDialogParameters& params
         {
             spdlog::warn( "GTK dialog failed" );
         }
+#if defined( __APPLE__ )
+        // on macOS the main window remains unfocused after the file dialog is closed
```

Comment:
Is this hack still actual with the native dialog?

Reply:
@oitel Yes. Otherwise user should select type first and then chose file.

With "*" instead ".*." user is able to chose any file (similar behaviour before this PR) and extension will be cheched.

Or user can select file type and only these files will be highlighted.